### PR TITLE
V2 models

### PR DESCRIPTION
**Pull request by alicefuzier**
### What is this PR trying to achieve?
Adding V2 models with contributors instead of creators
### Who is this change for?
🍔 🍨

### CODE REVIEW

File: catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/services/ElasticsearchServiceTest.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ class ElasticsearchServiceTest
         indexName = indexName,
         limit = 3,
         from = 0,
-        expectedWorks = List(work3, work2, work1).map { DisplayWork(_) }
+        expectedWorks = List(work3, work2, work1)
```
Comment (alexwlchan) [inline catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/services/ElasticsearchServiceTest.scala]: 👍 

oops

File: sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayContributionRole.scala [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package uk.ac.wellcome.display.models.v2
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+
+@ApiModel(
+  value = "ContributionRole",
+  description =
+    "A contribution role"
```
Comment (alicefuzier) [inline sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayContributionRole.scala]: I know this is probably not the correct documentation line we want to show. I couldn't find the schema for the contributors model so I'm going to chase with @silveroliver on that. For the time being, I'm putting in some deliberately basic descriptions

File: sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayContributionRole.scala [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+)
+case class DisplayContributionRole (
+                                     @ApiModelProperty(
+                                       value = "The name of the agent"
+                                     )
+                              label: String,
+  @JsonProperty("type") ontologyType: String = "ContributionRole"
+                              )
```
Comment (alexwlchan) [inline sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayContributionRole.scala]: IntelliJ has done wonders with the whitespace here. 😛

File: sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v1/DisplayWorkV1.scala [hunk 4]
```diff
@@ -57,7 +57,7 @@ case class DisplayWorkV1(
       "Relates a work to the general thesaurus-based concept that describes the work's content."
   ) subjects: List[DisplayConcept] = List(),
   @ApiModelProperty(
-    dataType = "uk.ac.wellcome.display.models.DisplayConcept",
+    dataType = "List[uk.ac.wellcome.display.models.DisplayConcept]",
```
Comment (alexwlchan) [inline sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v1/DisplayWorkV1.scala]: I’d double check, but we may be able to get rid of annotations where it’s a List[…] type – we override them in certain places if it’s an Option[…] or Option[List[…]] so the optional-ness doesn’t appear in the API, but the auto-generated values might be okay here.

File: sbt_common/display/src/test/scala/uk/ac/wellcome/display/models/v1/DisplayCreatorsV1SerialisationTest.scala [hunk 5]
```diff
@@ -8,7 +8,7 @@ import uk.ac.wellcome.display.modules.DisplayJacksonModule
 import uk.ac.wellcome.models._
 import uk.ac.wellcome.test.utils.JsonTestUtil
 
-class DisplayCreatorsSerialisationTest
+class DisplayCreatorsV1SerialisationTest
```
Comment (alexwlchan) [inline sbt_common/display/src/test/scala/uk/ac/wellcome/display/models/v1/DisplayCreatorsV1SerialisationTest.scala]: 👍 for naming

File: sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v1/DisplayWorkV1.scala (hunk 4 above)
Comment (alicefuzier) [reply sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v1/DisplayWorkV1.scala]: Uhm ok, I did have a quick look and we seem to override for lists everywhere, but it migh not be needed

**Pull request merged**